colosseum/wrappers.py

```diff
@@ -0,0 +1,48 @@
+from collections import Sequence
+
+
+class BorderSpacing:
+    """
+    Border spacing wrapper.
+
+    Examples:
+        BorderSpacing(1px)
+        BorderSpacing(1px, 2px)
+    """
+
+    def __init__(self, horizontal, vertical=None):
+        if isinstance(horizontal, Sequence) and not isinstance(horizontal, str):
+            raise TypeError('Invalid argument type for "horizontal"')
+
+        if isinstance(vertical, Sequence) and not isinstance(vertical, str):
+            raise TypeError('Invalid argument type for "vertical"')
```

Comment:
What's the value in including this type checking here? Won't this case be caught by the parser?

Reply:
Yeah, I should have erased this!

Reply:
Fixed!